docs/Users_Guide/gsi-tools.rst

```diff
@@ -311,7 +311,7 @@ Required Arguments for gsidens2orank
 
 1. The **ens_file_1 ... ens_file_n** argument is a list of ensemble binary GSI diagnostic files to be reformatted.
 
-2. The **ens_file_list** argument is an ASCII file containing a list of ensemble GSI diagnostic files.
+2. The **ens_file_list** argument is an ASCII file containing a list of ensemble GSI diagnostic files, as described in :numref:`ascii_file_lists`.
```

Comment:
Same suggestion here- change `ens_file_list` to be `file_list` and also in the usage code block for `gsi-tools`.

Reply:
Done in my feature branch.